### PR TITLE
Web Device API reference update for Tizen 2.3.2

### PR DESCRIPTION
**Pull request by pwasowski2**
Signed-off-by: Pawel Wasowski <p.wasowski2@partner.samsung.com>

### Change description ###

This commit updates and unifies style and formatting of Web Device API reference descriptions and code examples for Tizen 2.3.2.
On request of Mijin Cho, the commit also removes the API reference for mobile profile, since Tizen 2.3.2 did not support it.

### CODE REVIEW

File: docs/application/web/api/2.3.2/device_api/wearable/tizen/widgetservice.html [hunk 1]
```diff
@@ -475,31 +462,29 @@ <h4>Methods</h4>
               <li>
 AbortError - If the operation cannot be finished properly.              </li>
               <li>
-NotFoundError- If the device has no widgets or if a widget with the given ID does not exist.              </li>
+NotFoundError - If the device has no widgets or if a widget with the given id does not exist              </li>
```
Comment (mijin85cho) [inline docs/application/web/api/2.3.2/device_api/wearable/tizen/widgetservice.html]: Don't we need a period at the end of this sentence?

**Comment by mijin85cho** [comment]
Add one more issue : There are many "e.g."s in the files. Isn't it better to use "for example"?

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/413

**Comment by mijin85cho** [comment]
Hello, @vincent-kwon.
As you approved this update, 
may I just merge this PR without any update?

**Pull request merged**